java/ql/lib/ext/TopJdkApis.model.yml

```diff
@@ -0,0 +1,133 @@
+      - ["java.util", "List", "size", "()", "manual"]                     # UNSUPPORTED: type as sanitizer
+      - ["java.util", "List", "contains", "(Object)", "manual"]           # UNSUPPORTED: type as sanitizer
+      - ["java.util", "List", "isEmpty", "()", "manual"]                  # UNSUPPORTED: type as sanitizer
+      - ["java.util", "Map", "containsKey", "(Object)", "manual"]         # UNSUPPORTED: type as sanitizer
+      - ["java.util", "Map", "size", "()", "manual"]                      # UNSUPPORTED: type as sanitizer
+      - ["java.util", "Map", "isEmpty", "()", "manual"]                   # UNSUPPORTED: type as sanitizer
+      - ["java.util", "Optional", "isPresent", "()", "manual"]            # UNSUPPORTED: type as sanitizer
+      - ["java.util", "Optional", "empty", "()", "manual"]                # UNSUPPORTED: just gives empty Optional instance
+      - ["java.util", "UUID", "randomUUID", "()", "manual"]               # UNSUPPORTED: just gives UUID, no flow
+      - ["java.util", "UUID", "toString", "()", "manual"]                 # UNSUPPORTED: shouldn't model due to causing problems with dataflow?
+      - ["java.lang", "Object", "toString", "()", "manual"]               # UNSUPPORTED: shouldn't model due to causing problems with dataflow?
+      - ["java.lang", "Object", "equals", "(Object)", "manual"]           # **COLLISION** (both supported and unsupported per initial telemetry query): type as sanitizer
+      - ["java.lang", "Object", "getClass", "()", "manual"]               # UNSUPPORTED: only returns the class of the object, no dataflow?
+      - ["java.lang", "Class", "getName", "()", "manual"]                 # UNSUPPORTED: just returns class name, no flow as far as I can tell
+      - ["java.lang", "Class", "getSimpleName", "()", "manual"]           # UNSUPPORTED: just returns class name, no flow as far as I can tell
+      - ["java.lang", "String", "equals", "(Object)", "manual"]           # **SUPPORTED**: should be negative per Michael, but "supported" according to telemetry results (Note: no obvious MaD row... look into further, has the model been removed recently?)
+      - ["java.lang", "String", "equalsIgnoreCase", "(String)", "manual"] # UNSUPPORTED: type as sanitizer
+      - ["java.lang", "String", "length", "()", "manual"]                 # UNSUPPORTED: negative per Michael
+      - ["java.lang", "String", "isEmpty", "()", "manual"]                # UNSUPPORTED: type as sanitizer
+      - ["java.lang", "String", "contains", "(CharSequence)", "manual"]   # UNSUPPORTED: type as sanitizer
+      - ["java.lang", "String", "startsWith", "(String)", "manual"]       # UNSUPPORTED: type as sanitizer
+      - ["java.lang", "StringBuilder", "toString", "()", "manual"]        # **SUPPORTED**: shouldn't model due to causing problems with dataflow?, unless it's fine with StringBuilder? (Note: modelled in java.lang.model.yml at line 34 through AbstractStringBuilder.toString subtyping)
```

Comment:
This should not be in the negative models section, right? As you already note, it's already modelled.

Reply:
@yo-h had mentioned that `toString` should not be modeled because it causes problems with dataflow, which is why I put it with the "negative" models. Is `StringBuilder.toString` safe to model? Or should its existing model be removed and it turned into a negative model?

Reply:
`toString` in general, and particularly `Object.toString`, are unwanted models because they produce a lot of FP flow. But there are some classes which override `toString` in such a way that we're sure that they propagate taint, and `StringBuilder` is one of them. So we should keep the model and remove this row from the negative models.